src/dotnet-interactive.Tests/CommandLine/CommandLineParserTests.cs

```diff
@@ -174,6 +201,24 @@ public void http_command__does_not_parse_http_port_range_option()
                  .Contain(errorMessage => errorMessage == "Unrecognized command or argument '--http-port-range'");
         }
 
+        [Fact]
```

Comment:
The more interesting thing to test here is what type do I get when I resolve the `FrontendEnvironment`?

Reply:
none as the parser fails and we do not invoke it